src/compile/decode.js

```diff
@@ -205,11 +209,17 @@ var defaultValue = function (f, def) {
 
   switch (f.type) {
     case 'string':
-      return def != null ? def : ''
+      if (def) {
+        return def
```

Comment:
Is `def` the default value? Is it possible to have an empty string default value? If so we're returning `undefined` where we probably should be returning `''` here